packages/flutter_goldens/test/flutter_goldens_test.dart

```diff
@@ -153,6 +153,44 @@ void main() {
       );
     });
 
+    test('throws for error state from imgtestAdd', () {
+      final File goldenFile = fs.file('/workDirectory/temp/golden_file_test.png')
+        ..createSync(recursive: true);
+      platform = FakePlatform(
+          environment: <String, String>{
+            'FLUTTER_ROOT': _kFlutterRoot,
+            'GOLDCTL' : 'goldctl',
+          },
+          operatingSystem: 'macos'
```

Comment:
nit: add a trailing comma und unindent these lines by one?

Reply:
Ah yes thank you, will do